src/Engine/RadiumEngine.cpp

```diff
@@ -57,6 +58,12 @@ void RadiumEngine::initialize() {
 
     m_loadedFile.reset();
     Scene::ComponentMessenger::createInstance();
+
+    auto cameraManager = new Scene::DefaultCameraManager();
+    cameraManager->initialize();
+    Ra::Engine::RadiumEngine::getInstance()->registerSystem( "DefaultCameraManager",
```

Comment:
```suggestion
    Ra::Engine::RadiumEngine::getInstance()->registerSystem( "Camera Manager",
```

Reply:
This recall me that there is no `unregisterSystem` in the Engine class. If an application wants to change this camera manager and replace by its own ? How can this be done ?

Reply:
Good question, should we do this in the current PR ?

Reply:
I do not see any convention for system naming.
In current source code of Radium Lib, and plugins I found:
`
"GeometrySystem"
"DefaultLightManager"
"DefaultCameraManager"
"Minimal system"
"AnimationSystem"
"FakeToonSystem"
"DummySystem"
"MeshPaintSystem"
"LaplacianSmoothingSystem"
`
I propose to keep the current name for the moment until a convention is defined.